.github/CODEOWNERS

```diff
@@ -300,6 +300,7 @@
 
 
 # Navigation bar project.
+/core/templates/domain/bottom_navbar/ @nithusha21 @showtim3
```

Comment:
Also, just add Nithesh to codeowners as he is owning all the files related to this project!

Reply:
Done